Services/data/devicemodels/chiller-02.json

```diff
@@ -4,7 +4,7 @@
     "Version": "0.0.1",
     "Name": "Faulty Chiller",
     "Description": "Faulty chiller with wrong pressure sensor. Pressure too high.",
-    "Protocol": "MQTT",
+    "Protocol": "AMQP",
```

Comment:
Why is this changed to AMQP for only some of the device models and not all ?

Reply:
the others are already AMQP